base/applications/taskmgr/lang/fr-FR.rc

```diff
@@ -436,7 +436,7 @@ BEGIN
     IDS_MSG_UNABLEDEBUGPROCESS "Impossible de déboguer le processus."
     IDS_MSG_WARNINGDEBUG "ATTENTION : Déboguer ce processus peut entraîner une perte de données.\nÊtes-vous sûrs de vouloir le déboguer ?"
     IDS_MSG_TASKMGRWARNING "Avertissement du Gestionnaire des tâches"
-    IDS_MSG_WARNINGTERMINATING "ATTENTION : Terminer un processus peut causer des effets indésirables\nincluant une perte de donnée ou une instabilité du système.\nLe processus n'aura pas la chance de sauvegarder son état\nou les données avant de terminer.\nÊtes-vous sûr de vouloir terminer le processus ?"
+    IDS_MSG_WARNINGTERMINATING "ATTENTION : Terminer un processus peut engendrer la perte de donnée ou une instabilité du système.\nLe processus n'aura pas la chance de sauvegarder son état\nou ses données avant de terminer.\nVoulez-vous vraiment terminer le processus ?"
```

Comment:
```suggestion
    IDS_MSG_WARNINGTERMINATING "ATTENTION : Terminer un processus peut engendrer une perte de données ou une instabilité du système.\nLe processus n'aura pas la chance de sauvegarder son état\nou ses données avant de terminer.\nVoulez-vous vraiment terminer le processus ?"
```

Reply:
Since this suggestion is increasing the strings length by 2 chars again instead of shortening it, which was the actual goal of this PR, I will wait for a thumbs-up of Kyle katarn here whether that would be any substantial improvement before accepting Hermes suggestion.
If there won't be a thumbs up of Kyle, I will merge this as is in a week.

Reply:
Actually better. Thanks Hermès